ipatests/test_webui/test_user.py

```diff
@@ -408,3 +414,179 @@ def test_noprivate_gidnumber(self):
 
         self.add_record(user.ENTITY, user.DATA4, combobox_input='gidnumber')
         self.delete(user.ENTITY, [user.DATA4])
+
+
+@pytest.mark.tier1
+class TestLifeCycles(UI_driver):
+
+    def test_life_cycles(self):
+        """
+        Test user life-cycles
+        """
+
+        self.init_app()
+
+        # create "itest-user" and send him to preserved
+        self.add_record(user.ENTITY, user.DATA)
+        self.delete_record(user.PKEY, confirm_btn=None)
+        self.check_option('preserve', value='true')
+        self.dialog_button_click('ok')
+        self.assert_notification(assert_text='1 item(s) deleted')
+
+        # try to add the same user again (should fail)
+        self.add_record(user.ENTITY, user.DATA, assert_on=False)
+        err_dialog_txt = self.get_last_error_dialog().text
+        assert USR_EXIST.format(user.PKEY) in err_dialog_txt
```

Comment:
Could the new `assert_last_error_dialog` method be used here?

Reply:
fixed